packages/core/src/dispatcher.ts

```diff
@@ -39,6 +39,11 @@ const MAX_INTERVAL_WITHOUT_KEEPALIVE = 24 * 60 * 60 * 1000 // one day
 const IPFS_CACHE_SIZE = 1024 // maximum cache size of 256MB
 const IPFS_OFFLINE_GET_TIMEOUT = 200 // low timeout to work around lack of 'offline' flag support in js-ipfs
 const PUBSUB_CACHE_SIZE = 500
+const DEFAULT_RESPONSE_DELAY_MS = 1000 // 1 second
+const MINIMUM_RESPONSE_DELAY_MS = 0
+const MAXIMUM_RESPONSE_DELAY_MS = 16040 // 16 seconds and change
```

Comment:
we can reduce this - on average it will be this / # peers with the answer who are listening